bark/runtime/scenario/scenario_generation/config_readers/observer_model_config_readers.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2021 fortiss GmbH
+#
+# Authors: Julian Bernhard, Klemens Esterle, Patrick Hart and
+# Tobias Kessler
+#
+# This work is licensed under the terms of the MIT license.
+# For a copy, see <https://opensource.org/licenses/MIT>.
+
+
+import numpy as np
+import math
+
+from bark.runtime.scenario.scenario_generation.config_readers.config_readers_interfaces import ConfigReaderObserverModel
+from bark.runtime.commons.parameters import ParameterServer
+from bark.core.models.observer import ObserverModelNone, ObserverModelParametric
+
+
+# observed worlds with no perturbations
+class ObserverModelNoneReader(ConfigReaderObserverModel):
+  # returns a observer model none for the world
+  def create_from_config(self, config_param_object,  **kwargs):
+    params = ParameterServer()
```

Comment:
You should not create an own param server here, but just use the config param object. Have a look at the other readers how to pass down the param server

Reply:
yeah, that was a workaround